Cache/LoggingMemcachedInterface.php

```diff
@@ -12,11 +12,5 @@
  */
 interface LoggingMemcachedInterface
 {
-	/**
-	 * Get the logged calls for this Memcached object
-	 *
-	 * @return array Array of all of the calls made to the Memcached object
-	 */
-	public function getLoggedCalls();
```

Comment:
Should this documentation remain on the interface and just be inherited by anything implementing it?